services/PostService.js

```diff
@@ -0,0 +1,19 @@
+const Post = require('../models/Post');
+
+class PostService {
+  getOnePost(postId) {
+    return Post.find({ _id: postId });
+  }
+
+  getAllPosts() {
+    return Post.find({});
+  }
+
+  createNewPost(title, author, body) {
+    return Post({ title, author, body }).save();
```

Comment:
this should either be `return new Post({ title, author, body }).save();` or `return Post.create({ title, author, body });`

You shouldn't be calling `Post()` without a `new`

Reply:
fixed